app/models/Service.ts

```diff
@@ -47,8 +47,8 @@ export interface Service {
   status: "pending" | "approved" | "rejected" | "inactive";
   updated_at: string;
   url: string | null;
-  verified_at: any;
-  wait_time: any;
+  verified_at: Date | null;
+  wait_time: Date | null;
```

Comment:
Also 'fixes type for these fields based on api [db schema](https://github.com/ShelterTechSF/askdarcel-api/blob/02966597500d80d23701cf3d28ddce835d33340d/db/schema.rb#L322)